main.py

```diff
@@ -0,0 +1,29 @@
+from product_input_interface.input_interface import InputInterface
+
+
+class Main:
+    def __init__(self):
+        self.interface = InputInterface()
+
+    @staticmethod
+    def __print_choices():
+        print(
+            "Hello user. Choose what you want to do:\n"
+            "1. Add new product\n"
+            "0. Quit\n"
+        )
```

Comment:
that function is neither testable (only side-effects) nor re-usable (print the interface) - there is no point to make such functions instead of writing that code directly in line `18`. Currently code is harder to read because programmer must jump from function invocation `self.__print_choices()` to the implementation to understand what is going on